client/src/components/ListGroup/ListGroup.scss

```diff
@@ -55,11 +56,16 @@
 
 .list-group-item__thumbnail {
   width: 64px;
-  height: 64px;
+  height: auto;
   display: block;
   background: $gray-lighter;
   float: left;
-  margin: -12px 12px 0 #{0 - $grid-gutter-width / 2};
+  margin: -12px 12px -12px #{0 - $grid-gutter-width / 2};
+
+  > img {
+    max-height: 65px;
+    height: 100%;
```

Comment:
The thumbnails can end up looking like:

![silverstripe_-_campaigns](https://cloud.githubusercontent.com/assets/318116/25111816/60677cf4-2441-11e7-8b4a-32489c83d271.png)

And I think this is quite different from the design -
 https://projects.invisionapp.com/d/main#/projects/boards/1547737/71700347